governance.md

```diff
@@ -0,0 +1,129 @@
+
+# Projects
+
+The Confidential Containers organization is composed of multiple projects.
+
+A project is the primary unit of collaboration, therefore each project has its
+own repository and maintainers team. All projects follow the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+# Community Members and Roles
+
+Everyone is welcome to participate to the Confidential Containers projects,
+and there are 3 community member roles:
+
+## Contributor
+
+Anyone that contributed to one of the Confidential Containers projects within the
+last 12 months is a *Contributor*. Any merged Pull Request is considered a valid
+contribution.
+
+Contributions are not limited to code alone. Adding or enhancing documentation,
+tests, tools, project artifacts are all valuable ways to contribute to a
+Confidential Containers project.
+
+Project contributions will be reviewed by the project maintainers and should
+pass all applicable tests.
+
+## Maintainer
+
+Each project has one or more *Maintainer*. Project maintainers are first and
+foremost active *Contributors* to the project and are responsible for:
+
+* Setting technical directions for the project.
+* Facilitating, reviewing and merging contributions. They have write access to
+  the project repository.
+* Creating and assigning project issues.
+* Enforcing the [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md).
+
+The list of maintainers for a project is defined by the project
+`CODEOWNERS` file.
+
+### Becoming a project maintainer
+
+Existing maintainers may decide to elevate a *Contributor* to the *Maintainer*
+role based on the contributor established trust and contributions relevance.
+This decision process is not formally defined and is based on lazy consensus
+from the existing maintainers.
+
+Any contributor may request for becoming a project maintainer by opening an
+pull request (PR) against the `CODEOWNERS` file, and adding *all* current maintainers
+as reviewer of the PR.
+Maintainers may also pro-actively promote contributors based on their
+contributions and leadership track record.
+
+## Steering Committee Member
+
+The Steering Committee (SC) is the overall Confidential Containers organization
+governing body.
+
+The SC provides decision-making and strategic oversight for the project. It also
+defines and enforces the project values and structure.
+
+### Scope and Responsibilities
+
+The scope and responsibilites of the SC is subject to changes, and SC members must
+adapt it to meet the project needs.
+Moreover, any technical responsibilities should be delegated to project Maintainers,
+although the SC can be consulted to help the with making technical decisions.
+
+Based on that, the SC is responsible for:
+
+* Defining the project high-level strategy and roadmap
+* Managing and administrating the project, like e.g. preparing the weekly community meeting.
```

Comment:
Maybe another example here is actively looking to ensure there are always enough active maintainers for each project?